pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -333,6 +333,8 @@ func newRunnerPodForCR(ce chaosTypes.EngineInfo) (*corev1.Pod, error) {
 	//Initiate the Engine Info, with the type of runner to be used
 	if ce.Instance.Spec.Components.Runner.Type == "ansible" {
 		return newAnsibleRunnerPodForCR(ce)
+	} else if ce.Instance.Spec.Components.Runner.Type == "go" {
```

Comment:
Why this check is required? As we already have the check for ansible

Reply:
It is written to make sure that If anyone will look at the code, they can understand that it is only supporting go & ansible runners.
cc: @ksatchit